components/common/UserProfile/MainContent/Experiences/Experiences.types.ts

```diff
@@ -0,0 +1,18 @@
+export type ExperiencePayload = {
+  org_id: string;
+  title: string;
+  description: string;
+  start_at: string;
+  end_at: string;
+};
+
+export type Experience = ExperiencePayload;
+
+export type ExperiencesProps = {
+  orgId: string;
+  list: Experience[];
```

Comment:
```suggestion
  list: Array<Experience>;
```
I don't really have a preference either way, but we're using the generic style everywhere, should keep consistency